Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -1302,7 +1337,7 @@ def get_results_function_args(outputs, extended_data, item_type, interval_in_sec
 
 
 def pop_polling_related_args(args: dict) -> None:
-    for key in ('submit_file', 'enable_tor', 'interval_in_seconds', 'polling', 'environment_id'):
+    for key in ('submit_file', 'enable_tor', 'interval_in_seconds', 'environment_id'):
```

Comment:
The function `pop_polling_related_args` is called before requesting a status update when running the `upload file` command with polling, and we specifically need the `polling` argument, so we can know if we should throw an error or just a warning when analyzing the report.